### PR TITLE
Chappie1.0 styling inconsistent

### PR DESCRIPTION
**Pull request by svenjens**

### CODE REVIEW

File: apps/client/src/components/Answers.js [hunk 1]
```diff
@@ -2,19 +2,13 @@ import React from "react";
...
-//     padding-left: 15px;
-//   }
-
-//   .error-label {
-//     font-weight: 800;
-//     margin: 10px 0;
-//   }
+import { StyledAnswerErrorText } from "./AnswersStyles";
 
+const error = {
```
Comment (robinpiets) [inline apps/client/src/components/Answers.js]: Shall we change this to a Styled Component?

File: apps/client/src/components/AnswersStyles.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styled, { css } from "@datapunt/asc-core";
+
+export const StyledAnswer = styled(`div`)`
```
Comment (afjlambert) [inline apps/client/src/components/AnswersStyles.js]: can you write it like `styled.div`?

File: apps/client/src/components/AnswersStyles.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  ${props =>
+    props.hasError &&
+    css`
+      color: red;
+      border-left: 2px solid red;
+      padding-left: 15px;
+    `}
+`;
+
+export const StyledAnswerErrorText = styled(`div`)`
```
Comment (afjlambert) [inline apps/client/src/components/AnswersStyles.js]: same here

File: apps/client/src/components/AnswersStyles.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import styled, { css } from "@datapunt/asc-core";
+
+export const StyledAnswer = styled(`div`)`
+  ${props =>
+    props.hasError &&
+    css`
```
Comment (afjlambert) [inline apps/client/src/components/AnswersStyles.js]: why do you use `css` here?

File: apps/client/src/index.js [hunk 5]
```diff
@@ -22,6 +22,10 @@ import "@datapunt/asc-assets/lib/assets/fonts.css";
 
 dotenv.config();
 
+if (module.hot) {
+  module.hot.accept();
```
Comment (afjlambert) [inline apps/client/src/index.js]: Why is this required? Maybe add comment?

File: apps/client/src/components/AnswersStyles.js (hunk 4 above)
Comment (robinpiets) [reply apps/client/src/components/AnswersStyles.js]: That's the way to write conditionals in styled, right? https://styled-components.com/

File: apps/client/src/index.js (hunk 5 above)
Comment (robinpiets) [reply apps/client/src/index.js]: Added comment

File: apps/client/src/components/Layouts/BaseLayout.tsx [hunk 6]
```diff
@@ -27,9 +27,6 @@ function BaseLayout({ children, heading }: BaseLayoutProps) {
 
   return (
     <Container>
-      <Helmet>
```
Comment (afjlambert) [inline apps/client/src/components/Layouts/BaseLayout.tsx]: Why did you remove this?

File: apps/client/src/components/Nav.js [hunk 7]
```diff
@@ -51,7 +51,6 @@ const Nav = ({
           <Button
             type="submit"
             variant="secondary"
-            disabled={disableNext}
```
Comment (afjlambert) [inline apps/client/src/components/Nav.js]: Why did you remove this? Not sure if this is related to the findAddress part, if we have no valid postalCode a user should not be able to move forward. This makes form validation more complex imho.
ps. If disableNext is not used then it should be removed from the props as well and also remove the prop from references to this component.

File: apps/client/src/components/Layouts/BaseLayout.tsx (hunk 6 above)
Comment (robinpiets) [reply apps/client/src/components/Layouts/BaseLayout.tsx]: There should be no basic page title, as it's never called

File: apps/client/src/components/Nav.js (hunk 7 above)
Comment (robinpiets) [reply apps/client/src/components/Nav.js]: I would prefer not to use the disabled prop, because it makes the button invisible on the grey background (also not conform WCAG.) And other forms on the amsterdam.nl domain never disable the primary button and rely to form validation. I would prefer to do the same.
I will remove the `disableNext` prop.
image
